frontend/src/App.jsx

```diff
@@ -34,15 +39,23 @@ function App() {
   const handleLogout = () => {
     setIsLoggedIn(false);
     localStorage.setItem('isLoggedIn', 'false');
+    window.location.href = '/login';
```

Comment:
Use React Router's Hook, `useRouter` to handle navigation programmatically.

Reply:
Has been resolved